libs/common/src/auth/services/account.service.ts

```
@@ -213,7 +213,10 @@ export class AccountServiceImplementation implements InternalAccountService {
     );
   }
 
-  async setAccountVerifyDevices(userId: UserId, setVerifyNewDeviceLogin: boolean): Promise<void> {
+  async setAccountVerifyNewDeviceLogin(
+    userId: UserId,
+    setVerifyNewDeviceLogin: boolean,
+  ): Promise<void> {
     if (!Utils.isGuid(userId)) {
       // only store for valid userIds
       return;
```

Nit: we might want to throw an error here like other methods in this service do instead of returning silently. Would require updating the tests as well.